subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/verification/writer/WriteDependencyVerificationFile.java

```diff
@@ -167,10 +170,18 @@ public ModuleComponentRepository overrideDependencyVerification(ModuleComponentR
     public void buildFinished(Gradle gradle) {
         ensureOutputDirCreated();
         maybeReadExistingFile();
+        // when we generate the verification file, we intentionally ignore if the "use key servers" flag is false
+        // because otherwise it forces the user to disable the flag in the XML file, generate, then switch it back
```

Comment:
This _disable_ between two false reads weirdly.
Proposal:

> because otherwise it forces the user to remove the option in the XML file, generate, then switch it back